docs/develop/08_opendid/02_opendid_flow.md

```diff
@@ -0,0 +1,74 @@
+---
+id: flow
+title: OpenDID Flow
+---
+
+This guide explains the internal workings of OpenDID.
+Understanding this flow is helpful for setting up and configuring an OpenDID Service but less important if you only need to integrate it in an application.
+
+OpenDID includes interactions between multiple apps to authenticate and authorize users.
+Common use cases include the following:
+
+- Web app Frontend (app that includes the login button, for example, the demo app)
+- Web app back end
+- OpenDID front end
+- OpenDID back end
+- Identity wallet (a browser extension, for example, [Sporran](https://www.sporran.org/))
+
+The following steps outline the interactions necessary to implement the Implicit Flow:
+
+1. The user clicks the login button on the *web app front end*.
+2. The *web app front end* redirects the user to the *OpenDID front end*.
+3. The user chooses what wallet to authenticate with.
+4. The *OpenDID back end* establishes a secure session with the *identity wallet*.
+5. The *OpenDID back end* optionally requests a credential that implements a specific CType.
```

Comment:
I believe you can also filter by attester?

Reply:
I have no idea, @abdulmth ?

Reply:
Sorry I actually thought @abdulmth was the PR author. Yes, you can filter also by attester 😄

Reply:
True, the service will only accept credentials attested by a specific attester. But when you call `/credentials` only the CType will be returned. No information about the attester is returned.

Reply:
Ah ok, you are right.